File: code/backend/src/main/java/com/bycoders/cnab/dominio/entidades/TipoTransacao.java
package com.bycoders.cnab.dominio.entidades;

import javax.persistence.Entity;
import javax.persistence.Id;

import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode(callSuper=false)
@Entity
public class TipoTransacao extends PanacheEntityBase{
    @Id
    private Integer tipo;
    private String descricao;
    private String natureza;
    private String sinal;    

}

File: backEnd/src/Models/TipoTransacao.js
import Sequelize, { Model } from 'sequelize';


export default class TipoTransacao extends Model
{
	static init(sequelize)
	{
		super.init({
			tipo: Sequelize.INTEGER,
			descricao: Sequelize.STRING(30),
			natureza: Sequelize.STRING(20),
			sinal: Sequelize.STRING(2),
		},
			{
				sequelize,
				freezeTableName: true,
			});

		return this;
	}

}
